src/main/java/com/rapid7/ias/client/Configuration.java

```diff
@@ -15,7 +15,7 @@
 
 @javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2018-11-27T23:32:49.618-05:00")
 public class Configuration {
-    private static ApiClient defaultApiClient = new ApiClient();
+    private static ApiClient defaultApiClient = new ApiClient(null,null);
```

Comment:
Can probably just use the default constructor that you added rather than passing null?